### ISSUE TITLE
Cannot clone ssh with name starting with a dot

### ISSUE THREAD

**Issue by akiroz** [mssun/passforios#80]
I got the popup message:
```
Failed to clone repository from ________:akiroz/.password-store.git to file:///var/mobile/Containers/Data/Application/<A long ID>/Library/password-store-temp
```
Any idea how to find out what went wrong?

I'm using SSH key files generated from `ssh-keygen` from the `Download from URL` option.

**Comment by yishilin14** [comment]
Please double check our [SSH URL example](https://github.com/mssun/passforios/wiki#ssh-url-example).

I read your message, is it possible that you typed an extra "." before `password-store.git?`

**Comment by akiroz** [comment]
Oh didn't notice the wiki.

but still no luck after changing the URL to `ssh://git@_______/akiroz/.password-store.git`.
The leading `.` in the repo name is not a mistake, my local folder password store is a hidden folder.

**Comment by akiroz** [comment]
@yishilin14 
I managed to clone my repo after renaming my repo to `pass-test` (doesn't contain a leading `.`)
There might be a bug somewhere.....

**Comment by yishilin14** [comment]
That's interesting. I just created an empty repo with the leading `.`, and I was able to clone it. 

I will rename my testing password-store repo and do more testing.

**Comment by yishilin14** [comment]
@akiroz I just did some testing. 
- Authentication Method: Password + Clone a repo with name starting with a `.` --> OK
- Authentication Method: SSH Keys + Clone a repo with name starting with a `.` --> Failed

**Comment by asymmetric** [comment]
Still, I agree with the title of this issue, it would be nice to have some debug logs when cloning fails.

**Comment by mssun** [comment]
The fix was in the latest TestFlight. Can you try again? Thanks.

**Comment by yishilin14** [comment]
Fixed in v0.2.7 in app store, according to @mssun.
